compose-files/docker-compose.end-to-end-tests.yml

```diff
@@ -0,0 +1,47 @@
+version: "3"
+
+services:
+  server:
+    # env_file:
+    #   - ../envs/end-to-end-tests.env
+    command: npm run start
+
+  client:
+    # env_file:
+    #   - ../envs/end-to-end-tests.env
+    environment:
+      - DOCKER=true
+    command: npm run start
+
+  #nuxtclient:
+    # env_file:
+    #   - ../envs/end-to-end-tests.env
+
+  #notification-service:
+    #env_file:
+    #  - ../envs/end-to-end-tests.env
```

Comment:
why are these not enabled? Can the comments be removed?

Reply:
removed, is something we have to prove and should think about but not neccassery current